hummingbot/connector/exchange/bybit/bybit_order_book.py

```diff
@@ -0,0 +1,94 @@
+from typing import Dict, Optional
+
+from hummingbot.core.data_type.common import TradeType
+from hummingbot.core.data_type.order_book import OrderBook
+from hummingbot.core.data_type.order_book_message import OrderBookMessage, OrderBookMessageType
+
+
+class BybitOrderBook(OrderBook):
```

Comment:
Under the latest connectors' standards we should not create a special order book for the connector unless it is really required. In this case it is only used to have methods to create the even messages that are used in the order book data source. A special order book class is not required in this case.